.github/workflows/super-linter.yml

```
@@ -0,0 +1,21 @@
+---
+name: super-linter
+on:
+  pull_request:
+    branches: [main]
+  merge_group:
+  workflow_dispatch:
+jobs:
+  super-linter:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - name: Super-Linter
+        uses: super-linter/super-linter/slim@a8150b40c89574adb5f68bf9502b890a236a06b3 # v5.7.2
```

```suggestion
        uses: super-linter/super-linter/slim@v5.7.2
```

バージョン指定しないとCI `update-gitleaks` が正常に動かなそう。

---

あー了解